src/UriBuilder.Fluent/TerribleDevUriExtensions.cs

```diff
@@ -113,7 +165,7 @@ public static UriBuilder WithFragment(this UriBuilder bld, string key, IEnumerab
         /// <returns></returns>
         public static UriBuilder WithPort(this UriBuilder bld, int port)
         {
-            if(port < 1) throw new ArgumentOutOfRangeException(nameof(port));
+            if (port < 1) throw new ArgumentOutOfRangeException(nameof(port));
```

Comment:
I'm not in love with the reformatting of the code....but its fine to keep it

Reply:
Yeah sorry about that, VS decided to reformat the whole document. If you like, I can change my style rules back, reformat, and use the existing style rules that you had.